server/routes/ideaboard.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable consistent-return */
+const express = require('express');
+const ideaboard = require('../models/ideaboard.js');
+
+const router = express.Router();
+
+// index
+router.get('/', (req, res, next) => {
+  ideaboard.all((err, value) => {
+    if (err) return next(err);
+    res.json(value);
+  });
+});
+
```

Comment:
We probably want a route that gets all idea boards for a given user. Maybe even put it under the users route in `/users/:id/boards` which returns a list of board ids